crates/oxc_ast/src/syntax_directed_operations/bound_names.rs

```diff
@@ -135,7 +135,9 @@ impl<'a> BoundNames for FormalParameter<'a> {
 impl<'a> BoundNames for ModuleDeclaration<'a> {
     fn bound_names<F: FnMut(&BindingIdentifier)>(&self, f: &mut F) {
         if let ModuleDeclaration::ImportDeclaration(decl) = &self {
-            decl.bound_names(f);
+            if decl.import_kind.is_value() {
```

Comment:
Which is better?
1. Skip the type only import binding
2. Add `TypeAlias` as includes SymbolFlags (but it's not `TypeAlias`, `ImporType` or something elese is better)

Reply:
`BoundNames` should align with the spec, so skip the type only import binding?